packages/perseus/src/widgets/interactive-graphs/stateful-mafs-graph.tsx

```diff
@@ -15,9 +15,10 @@ import {getGradableGraph, getRadius} from "./reducer/interactive-graph-state";
 import type {InteractiveGraphProps, InteractiveGraphState} from "./types";
 import type {PerseusGraphType} from "../../perseus-types";
 import type {Widget} from "../../renderer";
+import type {APIOptions} from "../../types";
 
 export type StatefulMafsGraphProps = {
-    showLabelsFlag?: boolean;
+    flags?: APIOptions["flags"];
```

Comment:
I wonder if it would make sense to put the flags in a React context if we want to make them generally available to the various components. No action needed.

Reply:
YES! I wanted to do this during that one cooldown sprint, but never got around to it. I even think we have a [ticket for this](https://khanacademy.atlassian.net/browse/LEMS-2136), but it's marked as Won't Do.